.travis.yml

```diff
@@ -2,20 +2,20 @@ language: bash
 services: docker
 
 env:
-  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u3 VARIANT=slim
-  - VERSION=1.9 PYTHON_VERSION=3.5 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u3 VARIANT=slim
-  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=1.9 PYTHON_VERSION=3.5 PG_MAJOR=9.5 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=1.10 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=10+190.pgdg80+1 VARIANT=slim
-  - VERSION=1.10 PYTHON_VERSION=3.5 PG_MAJOR=9.6 PG_VERSION=10+190.pgdg80+1 VARIANT=slim
-  - VERSION=1.10 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=1.10 PYTHON_VERSION=3.5 PG_MAJOR=9.6 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=1.11 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=10+190.pgdg80+1 VARIANT=slim
-  - VERSION=1.11 PYTHON_VERSION=3.6 PG_MAJOR=9.6 PG_VERSION=10+190.pgdg80+1 VARIANT=slim
-  - VERSION=1.11 PYTHON_VERSION=2.7 PG_MAJOR=9.6 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=1.11 PYTHON_VERSION=3.6 PG_MAJOR=9.6 PG_VERSION=9.5.12-r0 VARIANT=alpine
-  - VERSION=2.0 PYTHON_VERSION=3.6 PG_MAJOR=9.6 PG_VERSION=10+190.pgdg80+1 VARIANT=slim
-  - VERSION=2.0 PYTHON_VERSION=3.6 PG_MAJOR=9.6 PG_VERSION=9.5.12-r0 VARIANT=alpine
+  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.4 PG_VERSION=9.4.18-2.pgdg90+1 VARIANT=slim
+  - VERSION=1.9 PYTHON_VERSION=3.6 PG_MAJOR=9.4 PG_VERSION=9.4.18-2.pgdg90+1 VARIANT=slim
```

Comment:
1.9 does not support Python 3.6, so I think that we should drop those items from the matrix (the image will still exist in Quay, it just won't get updated).

See: https://docs.djangoproject.com/en/2.0/faq/install/#what-python-version-can-i-use-with-django

Reply:
👍